include/utility/dbus_utility.hpp

```diff
@@ -58,6 +58,51 @@ inline types::MapperGetObject getObjectMap(const std::string& objectPath,
     return getObjectMap;
 }
 
+/**
+ * @brief An API to get property map for an interface.
+ *
+ * This API returns a map of property and its value of a particular
```

Comment:
nit: 
> of a particular


with respect to a particular interface.

Reply:
Done